### DISCUSSION TITLE
Integration with virtual display, resolution, swtich automations

### DISCUSSION THREAD

**Discussion by yujqiao** [LizardByte/community#536] [Sunshine]
### Select Topic Area

Question

### Body

I see lots of discussion around these popular scripts
https://github.com/itsmikethetech/Virtual-Display-Driver
https://github.com/Nonary/MonitorSwapAutomation
https://github.com/Nonary/ResolutionAutomation

It would be great if sunshine can integrate them.

A fork tried to do so but apparently not well maintained https://github.com/qiin2333/Sunshine-Foundation

**Comment by ReenigneArcher** [comment]
I guess you should encourage them to make a pull request instead of trying to maintain a fork?

**Comment by willow-sprite** [comment]
not sure about direct integration, but adding a setup tutorial to the documentation and a link to them would be nice.

on that note, an equivalent software to vdd for linux would be Extensible Virtual Display Interface.

https://github.com/DisplayLink/evdi

and saying somewhere in the documentation that simply plugging a second cable between your monitor and pc will give you a second screen you can share would also be nice.

[a comment by an automated account is not shown]
